tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -25,7 +25,8 @@
 #define LOG_MODULE_NAME bttester_gap
 LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
-#include "bttester.h"
+#include "btp.h"
+#include "btp_gap.h"
```

Comment:
Minor thing, but maybe it would be a good idea to place all the btp headers in common `btp` directory and just include all of them in `btp.h`. Check how it's done in `mesh.h` https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/bluetooth/mesh.h

Reply:
Done